packages/node_modules/@ciscospark/common/src/constants.js

```diff
@@ -1,16 +1,45 @@
-export const API_ACTIVITY_TYPE = {
-  REPLY: 'reply'
+export const WEBEX_EVENT = {
+  TEAMS_ACTIVITY: 'event:conversation.activity'
```

Comment:
is having a `.` in an event standard practice? I've seen the `:` and camelCase strings.

Reply:
This is the underlying conversations nomenclature that exists today.    We created the WEBEX_EVENT.TEAMS_ACTIVITY enums to abstract that a way a bit from the average SDK user/contributor